eclipse-scout-core/src/security/AccessControl.ts

```diff
@@ -25,16 +25,18 @@ export class AccessControl extends PropertyEventEmitter implements AccessControl
 
   protected _permissionCollection: PermissionCollection;
   protected _retryIntervals: number[];
-  protected _syncTimeoutId: number;
+  protected _permissionUpdateEventHandler: EventHandler<UiNotificationEvent>;
+  protected _reloadTimeoutId;
 
   constructor() {
     super();
     this.permissionsUrl = null;
-    this.interval = 1800000; // 30 * 60 * 1000 (30 minutes)
+    this.interval = 1_800_000; // 30 * 60 * 1000 (30 minutes)
```

Comment:
I think we should remove this.interval, because it does not reload every 30 min anymore. This variable now only specifies the limit for the retryIntervals array.
Actually, I don't think it should ever stop retrying, unless the session is invalidated.